diff --git a/ext/exif/config.w32 b/ext/exif/config.w32
--- a/ext/exif/config.w32
+++ b/ext/exif/config.w32
@@ -5,8 +5,4 @@
-if(PHP_EXIF != 'no')
-{
-	if(ADD_EXTENSION_DEP('exif', 'mbstring'))
-	{
-		AC_DEFINE('HAVE_EXIF', 1, "Define to 1 if the PHP extension 'exif' is available.");
-
-		EXTENSION('exif', 'exif.c', null, '/DZEND_ENABLE_STATIC_TSRMLS_CACHE=1');
-	}
+if(PHP_EXIF != 'no') {
+	AC_DEFINE('HAVE_EXIF', 1, "Define to 1 if the PHP extension 'exif' is available.");
+	EXTENSION('exif', 'exif.c', null, '/DZEND_ENABLE_STATIC_TSRMLS_CACHE=1');
+	ADD_EXTENSION_DEP('exif', 'mbstring', true);
